### PR TITLE
EWPP-1691: Set Event Programme field composite revisions true.

### PR DESCRIPTION
**Pull request by 22Alexandra**

### CODE REVIEW

**Comment by yenyasinn** [comment]
In the modules/oe_content_event/config/install/field.field.node.oe_event.oe_event_programme.yml need to set
```
third_party_settings:
  composite_reference:
    composite: true
    composite_revisions: true
```

File: modules/oe_content_event/config/install/field.storage.node.oe_event_programme.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ settings:
 module: entity_reference_revisions
 locked: false
 cardinality: -1
-translatable: false
+translatable: true
```
Comment (yenyasinn) [inline modules/oe_content_event/config/install/field.storage.node.oe_event_programme.yml]:
```suggestion
translatable: false
```
Actually field.field.node.oe_event.oe_event_programme.yml has `translatable: false`, so this field isn't translatable.

File: modules/oe_content_event/config/post_updates/20001_event_v2_fields/field.storage.node.oe_event_programme.yml [hunk 2]
```diff
@@ -14,7 +14,7 @@ settings:
 module: entity_reference_revisions
 locked: false
 cardinality: -1
-translatable: false
+translatable: true
```
Comment (yenyasinn) [inline modules/oe_content_event/config/post_updates/20001_event_v2_fields/field.storage.node.oe_event_programme.yml]:
```suggestion
translatable: false
```

File: modules/oe_content_event/config/install/field.storage.node.oe_event_programme.yml (hunk 1 above)
Comment (22Alexandra) [reply modules/oe_content_event/config/install/field.storage.node.oe_event_programme.yml]: Right

**Review by yenyasinn — DISMISSED** [review]

**Review by nagyad — DISMISSED** [review]

**Review by nagyad — DISMISSED** [review]

**Review by yenyasinn — DISMISSED** [review]

**Review by yenyasinn — APPROVED** [review]

**Review by nagyad — APPROVED** [review]

**Pull request merged**